packages/react-native-renderer/src/NativeMethodsMixin.js

```diff
@@ -161,7 +161,7 @@ export default function(
     measureLayout: function(
       relativeToNativeNode: number | Object,
       onSuccess: MeasureLayoutOnSuccessCallback,
-      onFail: () => void /* currently unused */,
+      onFail?: () => void /* currently unused */,
```

Comment:
This was always optional in the types used in React Native, but because the two types are duplicated in `ReactNativeTypes`, they must have fallen out of sync.